xsd/netex_framework/netex_reusableComponents/netex_nm_equipmentEnergy_support.xsd

```diff
@@ -188,6 +202,38 @@ Rail transport, Roads and Road transport
 		</xsd:simpleContent>
 	</xsd:complexType>
 	<!-- ======================================================================= -->
+	<xsd:simpleType name="FuelTypeListOfEnumerations">
+		<xsd:annotation>
+			<xsd:documentation>List of allowed values for file type.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:list itemType="FuelTypeEnumeration"/>
+	</xsd:simpleType>
+	<xsd:simpleType name="FuelTypeEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for type of fuel.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:normalizedString">
+			<xsd:enumeration value="battery"/>
+			<xsd:enumeration value="biodiesel"/>
+			<xsd:enumeration value="diesel"/>
+			<xsd:enumeration value="dieselBatteryHybrid"/>
+			<xsd:enumeration value="electricContact"/>
+			<xsd:enumeration value="electricity"/>
+			<xsd:enumeration value="ethanol"/>
+			<xsd:enumeration value="hydrogen"/>
+			<xsd:enumeration value="liquidGas"/>
+			<xsd:enumeration value="tpg"/>
```

Comment:
What is tpg?

Reply:
No idea (except Transports Publics Genevois  ;-) )

Reply:
Perhaps this was meant to be "lpg" - liquified petrol gas, also called autogas? It would be equivalent to liquidGas. lpg is stored at about 10 bar.
Then there is "cng" - compressed natural gas - stored at much higher pressure. A vehicle accepting cng works with naturalGas, bioGas (not in the list), and methane. Methane is the main ingredient of both naturalGas and bioGas.

There are still more problems, in my view. For a "dieselBatteryHybrid" vehicle, I would put all the following in *FuelType*: dieselBatteryHybrid, diesel, biodiesel, perhaps electricContact, electricity (but not battery, except if there is the possibility to replace an empty battery). 
Doesn't all this require some clarifications, rework, or annotations?

Reply:
If nobody knows what it is, deprecate it.